database/postgres/composition.yaml

```diff
@@ -43,6 +43,8 @@ spec:
             - type: string
               string:
                 fmt: "%s-postgresql"
+        - fromFieldPath: spec.writeConnectionSecretToRef.namespace
```

Comment:
@jbw976 should we set a default in the base or make this patch `Required` to guard against this value not being set on XR?

Reply:
good question @hasheddan.  there are a number of best practices and newer functionality that i'd like to get integrated into these reference platforms sometime soon (e.g. patchSets, required patches, etc.) in a complete and consistent way.

For now, I'm OK with following the same pattern already in place in this Configuration, e.g. here's another example in EKS: https://github.com/upbound/platform-ref-aws/blob/master/cluster/eks/composition.yaml#L71-L72, to minimize changes in this hot fix today.

I'll open an issue to track updating the reference platforms in a more complete way with all the latest 1.0/1.1 functionality and best practices :)